8.2.15 Release # DO NOT DELETE

su-sws-jira opened the pull request.

Commit:
feat(STN-129): Carousel Slick Controls Styling for Traditional and Colorful (#652)

* feat(STN-129): Carousel control styling for Traditional and Colorful

    - adds styles for the slick carousel arrows and dots for the traditional
      and colorful theme

* feat(STN-129): fininshing up Traditional carousel refinements

* feat(STN-129): accomodate for differences for hero in 2 col layout

* feat(STN-129): compiled CSS

su-sws-jira committed:
8.2.15

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr657-ri0xpkkhoaz5gucaboj9ggbzf9gtesy8.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f1773b62883883827e638bc

Visual Diffs:
- https://dashboard.tugboat.qa/5f1773b62883883827e638bc/visualdiffs

su-sws-jira committed:
Updated dependencies Jul 22 2020

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr657-ri0xpkkhoaz5gucaboj9ggbzf9gtesy8.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f1773b62883883827e638bc

Visual Diffs:
- https://dashboard.tugboat.qa/5f1773b62883883827e638bc/visualdiffs

joegl committed:
HSD8-864: Provisioned biologyvirtualshowcase (#659)

Co-authored-by: joegl <joe@joegl.com>

joegl committed:
HSD8-865: Provisioned history (#660)

Co-authored-by: joegl <joe@joegl.com>

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr657-ri0xpkkhoaz5gucaboj9ggbzf9gtesy8.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f1773b62883883827e638bc

Visual Diffs:
- https://dashboard.tugboat.qa/5f1773b62883883827e638bc/visualdiffs

Commit:
feat(STN-469):  hero opacity (#662)

* feat(STN-469): hero opacity changes

* chore(STN-469): compiled CSS

pookmish committed:
Fixed config ignore to ignore all enabled theme settings configs

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr657-ri0xpkkhoaz5gucaboj9ggbzf9gtesy8.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f1773b62883883827e638bc

Visual Diffs:
- https://dashboard.tugboat.qa/5f1773b62883883827e638bc/visualdiffs

Commit:
feat: added breakout box config and styles (#656)

* feat: added breakout box config and styles

- Added config for ckeditor types.
- Added styling for Humsci Basic themes.
- Added styling for Humsci Legacy theme.

* update: remove align top margin, add display table

* chore: update comment for hb-well class

* chore: compiled css

* chore: fix stylelint errors

Co-authored-by: Kasey Bonifacio <kasey@heysparkbox.com>

Commit:
feat(STN-459): ensure Spotlight works on legacy theme (#654)

* feat(STN-459): bring back Spotlight paragraph type to codebase

- This reverts commit 50a8c28294132089ce71b9cbcf01818bf18c1506 which was
  accomplished in PR (#647)

* feat(STN-459): adds styles for Su_humsci Spotlight

* fix(STN-459): linting fixes

* refactor(STN-459): rename display to height for Spotlight field

* chore(STN-459): package-lock update

* feat(STN-430): Spotlight design update/enhancements (#658)

* feat(430): spotlight design enhancements

* feat(STN-430): further spotlight enhancements and refactor

* chore(STN-430): compiled CSS

* feat(STN-459): width adjustment to text and image

* fix(STN-459): Spotlight padding

kaseybon committed:
chore: compiled CSS

pookmish committed:
Sparkbox Sprint 21

pookmish committed:
HSD8-852 Map 3 more field from cap

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr657-ri0xpkkhoaz5gucaboj9ggbzf9gtesy8.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f1773b62883883827e638bc

Visual Diffs:
- https://dashboard.tugboat.qa/5f1773b62883883827e638bc/visualdiffs

Commit:
fix(STN-434): implements both a timeout and promise behavior to try to ensure the stretch cards load correctly (#663)

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr657-ri0xpkkhoaz5gucaboj9ggbzf9gtesy8.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f1773b62883883827e638bc

Visual Diffs:
- https://dashboard.tugboat.qa/5f1773b62883883827e638bc/visualdiffs

Commit:
fix(STN-487): .hs-well and Traditional Theme long arrow svg (#670)

* fix(STN-487): color issue with traditional arrows

* fix(STN-489): display table only on hs-well in wysiwyg areas (#669)

* chore(STN-487 and STN-489): compiled CSS

joegl committed:
HSD8-870 Provisioned humanexperience (#674)

Co-authored-by: joegl <joe@joegl.com>

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr657-ri0xpkkhoaz5gucaboj9ggbzf9gtesy8.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f1773b62883883827e638bc

Visual Diffs:
- https://dashboard.tugboat.qa/5f1773b62883883827e638bc/visualdiffs
